oms-storage/docker/nextcloud-nginx.conf

```diff
@@ -0,0 +1,168 @@
+worker_processes auto;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
```

Comment:
check out what I did in oms-events and oms-frontend with set_real_ip, otherwise it'd log docker IP instead of the real one. Might be helpful for kibana.

Reply:
specifically https://github.com/AEGEE/oms-events/blob/f4beeed8915c541bbfdf6b05efaac6acca512ee6/docker/oms-events-static/nginx.conf#L28

Reply:
fantastic, thanks